README.md

```diff
@@ -18,14 +18,18 @@ Use this plugin to improve onboarding and HR processes. It adds a Welcome Bot th
 
 ## Configuration
 
-1. Go to **System Console > Plugins > Management** and click **Enable** to enable the Welcome Bot plugin.
+1. Go to **System Console > Plugin Management** and click **Enable** to enable the Welcome Bot plugin.
```

Comment:
```suggestion
1. Go to **System Console > PLUGINS > Plugin Management** and click **Enable** to enable the Welcome Bot plugin.
```

`Plugins` is the parent section name for `Plugin Management`.  This is a small change and I'm 0/5 whether it should be included, but think it is more clear on how to find the `Pluigin Management`.

Reply:
Nice improvement, @jwilander.  I only have one suggestion with regard to the README.